webapp/i18n/en.json

```
@@ -246,51 +283,31 @@
   "ViewTitle.untitled-board": "Untitled board",
   "WelcomePage.Description": "Boards is a project management tool that helps define, organize, track and manage work across teams, using a familiar kanban board view",
   "WelcomePage.Explore.Button": "Take a tour",
-  "WelcomePage.NoThanks.Text": "No thanks, I'll figure it out myself",
   "WelcomePage.Heading": "Welcome To Boards",
+  "WelcomePage.NoThanks.Text": "No thanks, I'll figure it out myself",
   "Workspace.editing-board-template": "You're editing a board template.",
   "calendar.month": "Month",
   "calendar.today": "TODAY",
   "calendar.week": "Week",
+  "cloudMessage.learn-more": "Learn more",
   "default-properties.badges": "Comments and Description",
   "default-properties.title": "Title",
-  "error.relogin": "Log in again",
-  "login.log-in-button": "Log in",
-  "login.log-in-title": "Log in",
-  "error.workspace-undefined": "Not a valid workspace.",
-  "error.page.title": "Sorry, something went wrong",
-  "error.not-logged-in": "Your session may have expired or you're not logged in.",
+  "error.back-to-boards": "Go to the Dashboard",
   "error.back-to-home": "Back to Home",
-  "error.back-to-boards": "Back to boards",
-  "error.go-login": "Log in",
+  "error.go-login": "Login",
```

When you re-ran the extract, it pulled from default. I believe this will make cypress changes unnecessary. See original translation. 

```suggestion
  "error.go-login": "Log in",
```

---

Yep, you could do that as well. The default and translation have diverged. You can change the default and re-run `i18n-extract` or change the translation. The bottom line is, the button text shouldn't change.

https://github.com/mattermost/focalboard/blob/release-7.0/webapp/i18n/en.json#L265